lib/elixir/lib/calendar.ex

```diff
@@ -2314,6 +2336,21 @@ defmodule DateTime do
     {:ok, result_datetime}
   end
 
+  @doc """
+  Converts a DateTime from one calendar to another.
+  If this conversion fails for some reason, an ArgumentError is raised.
```

Comment:
Please add a new line to start a new paragraph. The first paragraph must always be a short summary.